### PR TITLE
Fix/901 2016 2019 properties

### PR DESCRIPTION
**Pull request by mdahlhausen**

### CODE REVIEW

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2016/data/ashrae_90_1_2016.prototype_inputs.json [hunk 1]
```diff
@@ -229,7 +229,7 @@
       "pszac_operation_schedule_3": null,
       "pszac_oa_damper_schedule_3": null,
       "occ_sensing_exterior_lighting_power": 1459.5,
-      "occ_sensing_exterior_lighting_schedule": "RestaurantSitDown Exterior_lighting_schedule_b",
+      "occ_sensing_exterior_lighting_schedule": "RestaurantSitDown Exterior_lighting_schedule_b_2016",
```
Comment (lymereJ) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2016/data/ashrae_90_1_2016.prototype_inputs.json:232]: @leijerry888 - Is that a correct change? From [this PR](https://github.com/NREL/openstudio-standards/pull/1055/files) it shows that `Exterior_lighting_schedule_b` is correct.

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2019/data/ashrae_90_1_2019.prototype_inputs.json [hunk 2]
```diff
@@ -2065,7 +2065,7 @@
       "building_type": "SecondarySchool",
       "hvac_system": null,
       "geometry_osm": "ASHRAESecondarySchool.osm",
-      "hvac_json": "ASHRAE90120162019SecondarySchool.json",
+      "hvac_json": "ASHRAE20162019SecondarySchool.json",
```
Comment (lymereJ) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2019/data/ashrae_90_1_2019.prototype_inputs.json:2068]: #1060 used "ASHRAE90120162019SecondarySchool.json" which is what is currently on the master branch of the repository (see [here](https://github.com/NREL/openstudio-standards/blob/master/data/geometry/ASHRAE90120162019SecondarySchool.json)), not sure if this is a valid change.

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2019/data/ashrae_90_1_2019.prototype_inputs.json [hunk 3]
```diff
@@ -932,7 +932,7 @@
       "building_type": "LargeOffice",
       "hvac_system": null,
       "geometry_osm": "ASHRAE9012013LargeOffice.osm",
-      "hvac_json": "ASHRAE90120162019LargeOffice.json",
+      "hvac_json": "ASHRAE20162019LargeOffice.json",
```
Comment (lymereJ) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2019/data/ashrae_90_1_2019.prototype_inputs.json:935]: #1060 used "ASHRAE90120162019LargeOffice.json" which is what is currently on the master branch of the repository (see [here](https://github.com/NREL/openstudio-standards/blob/master/data/geometry/ASHRAE90120162019LargeOffice.json)), not sure if this is a valid change.

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2019/data/ashrae_90_1_2019.prototype_inputs.json [hunk 4]
```diff
@@ -829,7 +829,7 @@
       "building_type": "LargeHotel",
       "hvac_system": null,
       "geometry_osm": "ASHRAELargeHotel.osm",
-      "hvac_json": "ASHRAE90120162019LargeHotel.json",
+      "hvac_json": "ASHRAE20162019LargeHotel.json",
```
Comment (lymereJ) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2019/data/ashrae_90_1_2019.prototype_inputs.json:832]: Same comment as below.

**Comment by mdahlhausen** [comment]
@lymereJ can you update the regression tests?

**Pull request merged**